package.json

```diff
@@ -20,11 +20,12 @@
   "author": "Augusto Lemble <me@augustolemble.com>",
   "license": "ISC",
   "dependencies": {
+    "abi-decoder": "^1.0.8",
     "ethereumjs-abi": "^0.6.4",
     "protobufjs": "^6.0.2",
     "solc": "^0.4.9",
     "truffle-hdwallet-provider": "0.0.3",
-    "zeppelin-solidity": "^1.0.4"
+    "zeppelin-solidity": "1.0.4"
```

Comment:
@AugustoL no, it's ok, let's keep it as is. in this case, they are small changes, so they don't even make the review much more difficult. I wanted to mention it thinking in future PRs, and also to have this discussion :)